python/src/Contour_doc.i.in

```diff
@@ -47,8 +50,36 @@ Examples
 >>> # Create the contour
 >>> myContour = ot.Contour(nX + 2, nY + 2, data)
 >>> myContour.setLevels(levels)
+>>> myGraph.add(myContour)
+
+Using the second constructor:
+
+>>> import openturns as ot
+>>> g = ot.SymbolicFunction(['R','S'],['(R * S)^0.75'])
+>>> # Generate the data for the curves to be drawn
+>>> nX = 10
+>>> nY = 10
+>>> lowerBound = ot.Point([1.6, 0.0])
+>>> upperBound = ot.Point([6.3, 4.3])
```

Comment:
no need to use Point here, just plain lists

Reply:
Done.